internal/cloud/realm/dependencies.go

```diff
@@ -18,10 +18,44 @@ const (
 	dependenciesPathPattern        = appPathPattern + "/dependencies"
 	dependenciesArchivePathPattern = dependenciesPathPattern + "/archive"
 	dependenciesDiffPathPattern    = dependenciesPathPattern + "/diff"
+	dependenciesStatusPathPattern  = dependenciesPathPattern + "/status"
 
 	paramFile = "file"
 )
 
+// DependenciesStatus is used to get information from a dependencies status request
+type DependenciesStatus struct {
+	State   string `json:"status"`
```

Comment:
I think this may be the best way to avoid stutter but I can see how the idea of a State of Status is a bit odd.